app/src/main/java/kr/co/connect/boostcamp/livewhere/firebase/ReviewMapper.kt

```diff
@@ -0,0 +1,19 @@
+package kr.co.connect.boostcamp.livewhere.firebase
+
+import kr.co.connect.boostcamp.livewhere.model.Review
+import kr.co.connect.boostcamp.livewhere.model.entity.ReviewEntity
+
+class ReviewMapper : FirebaseMapper<ReviewEntity, Review>() {
+
+    override fun map(reviewEntity: ReviewEntity): Review {
+        val review = Review()
+        review.id = reviewEntity.id
+        review.nickname = reviewEntity.nickname
+        review.contents = reviewEntity.contents
+        review.land_code = reviewEntity.land_code
+        review.date = reviewEntity.date
+        return review
```

Comment:
`p2;` data class 생성자 호출 시 다음과 같이 넣어주는 것이 나아보입니다. var키워드 사용은 지양해주세요.
```suggestion
        return Review(
          id = reviewEntity.id
          nickname = reviewEntity.nickname
          contents = reviewEntity.contents
          land_code = reviewEntity.land_code
          date = reviewEntity.date
        )
```